deployments/vanilla/terraform/main.tf

```diff
@@ -62,22 +115,13 @@ module "eks_blueprints" {
   vpc_id             = module.vpc.vpc_id
   private_subnet_ids = module.vpc.private_subnets
 
-  managed_node_groups = {
-    mg_5 = {
-      node_group_name = "managed-ondemand"
-      instance_types  = ["m5.large"]
-      min_size        = 5
-      desired_size    = 5
-      max_size        = 10
-      subnet_ids      = module.vpc.private_subnets
-    }
-  }
-
+  # configuration settings: https://github.com/aws-ia/terraform-aws-eks-blueprints/blob/main/modules/aws-eks-managed-node-groups/locals.tf
+  managed_node_groups = local.managed_node_groups
   tags = local.tags
 }
 
 module "eks_blueprints_kubernetes_addons" {
-  source = "github.com/aws-ia/terraform-aws-eks-blueprints//modules/kubernetes-addons?ref=v4.9.0"
+  source = "github.com/aws-ia/terraform-aws-eks-blueprints//modules/kubernetes-addons?ref=v4.12.0"
```

Comment:
this change this missing for other deployment options. can we update the tag for everything like eks_blueprints module? its better to keep them in sync